applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py

```diff
@@ -303,7 +303,7 @@ def compose(self):
     parser.add_argument(
         "-d",
         "--data",
-        default="none",
+        default=os.environ.get("HOLOSCAN_INPUT_PATH", None),
```

Comment:
Seems like this should be the data path?

```suggestion
        default=os.environ.get("HOLOSCAN_DATA_PATH", None),
```

Reply:
It is `HOLOSCAN_INPUT_PATH` per [HAP specification](https://docs.nvidia.com/holoscan/sdk-user-guide/cli/hap.html#table-of-environment-variables)